docs/integration.md

```diff
@@ -0,0 +1,248 @@
+# [Integration Guide](#integration-guide)
+The basis of this document is to chowcase the process of integrating `bridget` into an existing web delivery. The end-goal of this module is adding bridging support to the website when browsed through the context of a mobile app via a WebView.
+
+## [Prerequisites](#prerequisites)
+
+This integration guide assumes you already have a web delivery that serves content & a mobile app is being setup for that  particular web delivery.
+
+## [Requirements](#requirements)
+
+#TODO - write requirements that the web delivery needs to satisfy in order for the module to work
```

Comment:
unclear to me what kind of content should go here... maybe not write todos until it is ready